src/Notification/Hydrator/NotificationHydrator.php

```diff
@@ -39,6 +40,23 @@ public function hydrate(Notification $notification): NotificationListItem
         );
     }
 
+    public function hydrateDetail(NotificationModel $notification): Notification
+    {
+        return new Notification(
+            $notification->getId(),
+            $notification->getType(),
+            $notification->getTitle(),
+            $notification->isRead(),
+            (bool)$notification->getLinkedElementType(),
+            (new Carbon($notification->getCreationDate(), 'UTC'))->getTimeStamp(),
```

Comment:
Should we add Carbon to the composer requirements? Since it is used by the Bundle, to be consistent. 
WDYT?

Reply:
Not sure if we have here any Rules here. Or if it is okay when it is requireds by the core.

Reply:
Yeah, good question. I mean it does not work without the core, but the bundle itself, should also list its requirements. So I would add it tbh.

Reply:
I will add it then :)